packages/core/src/Stores/client-store.js

```diff
@@ -505,13 +505,32 @@ export default class ClientStore extends BaseStore {
     }
 
     get can_change_fiat_currency() {
-        const has_no_mt5 = !this.has_real_mt5_login;
-        const has_no_dxtrade = !this.has_real_dxtrade_login;
+        const eu_user = this.root_store.traders_hub.is_eu_user;
+
+        const has_maltainvest_mt5_account = this.mt5_login_list.some(
+            acc => acc.landing_company_short === 'maltainvest'
+        );
+        const has_svg_and_financial_mt5_account = this.mt5_login_list.some(
+            acc => acc.landing_company_short === 'svg' || acc.landing_company_short === 'financial'
+        );
+
+        const no_mt5_non_eu = !eu_user && !has_svg_and_financial_mt5_account;
+
+        const no_eu_mt5 = eu_user && !has_maltainvest_mt5_account;
+
+        const has_no_dxtrade = this.dxtrade_accounts_list.some(acc => acc.landing_company_short === 'svg') && !eu_user;
```

Comment:
Eu users cannot trade using dxtrade. DXTrade option is not available for them and they cannot create an account. "is_eu" is redundant here.

Reply:
Will remove it @hirad-rewok 🙏